resources/shaders/lighting.hlsl

```diff
@@ -154,8 +155,20 @@ float4 DoShadowAllLights(float3 wpos, float3 V, float3 normal, float metallic, f
 		float light_dist = length(L);
 		L /= light_dist;
 
+		float dir_dot = dot(L, normal);
```

Comment:
Not sure what the style guide says but this is not a propper abbreviation could mean multiple things. The fact that it's clear from the context is beside the point here